cmd/rekor-server/app/root.go

```diff
@@ -115,6 +115,7 @@ Memory and file-based signers should only be used for testing.`)
 	rootCmd.PersistentFlags().String("redis_server.address", "127.0.0.1", "Redis server address")
 	rootCmd.PersistentFlags().Uint16("redis_server.port", 6379, "Redis server port")
 	rootCmd.PersistentFlags().String("redis_server.password", "", "Redis server password")
+	rootCmd.PersistentFlags().Bool("redis_server.insecure-skip-verify", false, "Whether to skip TLS verification or not")
```

Comment:
nit: I'd add something in the description to note that this is only for connections to Redis

Reply:
also, setting this default to `false` would represent a breaking change for clients (which all would be connecting without TLS today) - is that correct?

Reply:
Yes, it would be a breaking change, we can change it to 'true', but that feels like going against the standard approach when adding flags of this nature where you would expect an explicit opt-out from a more 'secure' way, open to feedback on this though, might just be my understanding of it. I pushed a commit to change the description to mention that this applies to the redis endpoint, not sure if it's clear enough.

Reply:
I think we'd just need to be sure the next release bumped the minor version of Rekor with a clear note that if folks are using Redis that they'd need to specify the flag - tagging @haydentherapper for his thoughts

Reply:
I wouldn't include insecure in the flag name and instead call this "enable-tls" or something similar, which would avoid a breaking change. I don't think "insecure" needs to be mentioned because skipping TLS verification is not necessarily insecure depending on the setup - a redis instance could be behind a VPC in a production setup for example.

Reply:
If we rename skipVerify to enable-tls then disabling the breaking change would mean that we would need to set the default values to 'true' for it. That seems a bit odd. InsecureSkipVerify is a pretty standard way of defining this feature and widely adopted.

Reply:
Sorry, I'm not following how adding an enable-tls flag would be a breaking change. Given this feature is adding support for TLS, we could add a flag for enabling TLS set to false by default, and for those who need TLS, they can set the new flag to true. This is just the inverse of the current proposal to have an insecure flag set to true by default, it seems like better optics to have an enable flag set to false rather than an insecure flag set to true.

Reply:
Sorry, misunderstood, was under the impression we were still only talking about the insecureFlag and not TLS in general. 

Pushed a commit ( https://github.com/sigstore/rekor/pull/1998/commits/53238d4b0c708ddf987a538d85d2310614672651 ) that introduces the 'enable-tls' flag under the 'redis_server' group ( in addition to InsecureSkipVerify - Added explanation to the description that mentions it's only applicable when redis enable-tls is set ).

Had to extract the client creation into a separate function to wrap it with an if statement based on the 'enable-tls' flag value, not sure if that is the best way to do this.